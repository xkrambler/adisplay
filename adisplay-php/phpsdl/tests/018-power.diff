010+ bool(false)
011+ bool(false)
010- bool(true)
011- bool(true)