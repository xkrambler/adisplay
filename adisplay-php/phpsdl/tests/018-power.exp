= Constants
bool(true)
bool(true)
bool(true)
bool(true)
bool(true)
= Functions
bool(true)
bool(true)
bool(true)
bool(true)
= Done