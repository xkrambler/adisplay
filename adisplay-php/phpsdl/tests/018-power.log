
---- EXPECTED OUTPUT
= Constants
bool(true)
bool(true)
bool(true)
bool(true)
bool(true)
= Functions
bool(true)
bool(true)
bool(true)
bool(true)
= Done
---- ACTUAL OUTPUT
= Constants
bool(true)
bool(true)
bool(true)
bool(true)
bool(true)
= Functions
bool(true)
bool(true)
bool(false)
bool(false)
= Done
---- FAILED
