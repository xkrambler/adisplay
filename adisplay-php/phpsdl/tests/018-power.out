= Constants
bool(true)
bool(true)
bool(true)
bool(true)
bool(true)
= Functions
bool(true)
bool(true)
bool(false)
bool(false)
= Done